README.md

```diff
@@ -35,13 +37,9 @@ Compile assets using [Webpack Encore](https://symfony.com/doc/4.4/frontend.html)
 
 `npm run watch`
 
-Run local Datastore emulator
+Run docker container
 
-`gcloud beta emulators datastore start`
-
-Run local App Engine dev server:
-
-`./bin/console pmi:deploy --local`
+`docker-compose up`
```

Comment:
```suggestion
`docker compose up`
```

(compose v1 is now [deprecated](https://www.docker.com/blog/announcing-compose-v2-general-availability/))